library/std/src/os/fd/raw.rs

```diff
@@ -40,10 +40,7 @@ pub trait AsRawFd {
     /// ```no_run
     /// use std::fs::File;
     /// # use std::io;
-    /// #[cfg(unix)]
-    /// use std::os::unix::io::{AsRawFd, RawFd};
-    /// #[cfg(target_os = "wasi")]
-    /// use std::os::wasi::io::{AsRawFd, RawFd};
+    /// use std::os::fd::{AsRawFd, RawFd};
```

Comment:
This shouldn't compile, if the stability attributes work correctly.

Reply:
It does compile, somehow. The doc tests pass. I don't understand stability attributes enough to know what's going on.